lago/prefix.py

```diff
@@ -1225,6 +1226,7 @@ def _deploy_host(self, host):
                 return
 
             with LogTask('Wait for ssh connectivity'):
+                time.sleep(5)
```

Comment:
OK, so the caller of env_start essentially will do the sleep, before calling env_deploy?